microcosm_daemon/state_machine.py

```diff
@@ -10,9 +11,10 @@ class StateMachine(object):
 
     """
 
-    def __init__(self, graph, initial_state):
+    def __init__(self, graph, initial_state, never_reload=False):
```

Comment:
I though it'd be nice to support disabling reloading, but perhaps this is YAGNI?